Fundsoft/report.md

```diff
@@ -100,6 +100,20 @@ Scope (functionality and code affected).
 
 Optional (point 3): trace tests to requirements.
 
+* test_update_transitive()
+    * This test adds the dependency `requests` and later updates the sub-dependency `chardet`. Asserts that the sub-dependency is still not in the lock file and that the sub-dependency has been updated to the newest version from the virtual repository. Also asserts that the dependency `requests` has not been updated as this was not requested. 
```

Comment:
The test checks that the dependency is not in the pyproject.toml file. All sub-dependencies will still be in the lock file.